### ISSUE TITLE
check JDL files at VM or other means of job counting

### ISSUE THREAD

**Issue by vmendez** [DIRACGrid/VMDIRAC#88]
using VMmonitor
This could be an issue if jdl files are not found at VM, because of github commit made by Andrei:
https://github.com/DIRACGrid/DIRAC/commit/0c82f4ffe29dd6f86ddd351ecaba0f1297b3cf3f

**Comment by vmendez** [comment]
related with #66

**Comment by myco** [comment]
Any ideas when it be fixed?
I see that on EGI WMS  is version of DIRAC with old JobAgent.py so that VMDIRAC can works so far...
What about simply revert this commit?
 https://github.com/DIRACGrid/DIRAC/commit/0c82f4ffe29dd6f86ddd351ecaba0f1297b3cf3f

**Comment by atsareg** [comment]
I can easily revert that. But I would like to understand how the JDL of the job is used by VMDIRAC ?

**Comment by myco** [comment]
On VM, VirtualMachineMonitorAgent is counting JDL files of job wrappers and send this number to  VirtualMachineManager -> DB, than we can see in web portal how many jobs has been processed on particular VM.

**Comment by myco** [comment]
I see the fix https://github.com/DIRACGrid/DIRAC/commit/d607dd10498e1840da0d2bc5982e913fd4af0b67
but it was applied to DIRAC version 7 only.
Please apply this commit to v6r12 and v16r13 as well.
Than this issue can be resolved.
Thank you !

**Comment by vmendez** [comment]
Hi Andrei,

¿May you apply the changes in the releases asked by Rafal, please?

Víctor

2015-02-17 18:05 GMT+01:00 myco notifications@github.com:

**Comment by atsareg** [comment]
```
  Done in v6r12p21

    Andrei

  On 18/02/15 12:54, Víctor Méndez wrote:

Hi Andrei,

  ¿May you apply the changes in the releases asked by Rafal, please?

  Víctor

  2015-02-17 18:05 GMT+01:00 myco <notifications@github.com>:

  > I see the fix DIRACGrid/DIRAC@d607dd1
  >
```

https://github.com/DIRACGrid/DIRAC/commit/d607dd10498e1840da0d2bc5982e913fd4af0b67
      > but it was applied to DIRAC version 7 only.
      > Please apply this commit to v6r12 and v16r13 as well.
      > Than this issue can be resolved.
      > Thank you !
      >
      > —
      > Reply to this email directly or view it on GitHub
      >
https://github.com/DIRACGrid/VMDIRAC/issues/88#issuecomment-74705338.
      >
      —
        Reply to this email directly or view
          it on GitHub.
